src/support/azext_support/_validators.py

```diff
@@ -134,10 +135,31 @@ def _check_name_availability_no_subscription(cli_ctx, resource_name, resource_ty
     if not resp["nameAvailable"]:
         raise CLIError(resp["message"])
 
-
 def _check_name_availability_no_subscription_ticket(cli_ctx, ticket_name, resource_name, resource_type):
     from .aaz.latest.support.no_subscription.tickets import CheckNameAvailability
     check_name_availability_input = {"support_ticket_name": ticket_name, "name": resource_name, "type": resource_type}
     resp = CheckNameAvailability(cli_ctx=cli_ctx)(command_args=check_name_availability_input)
     if not resp["nameAvailable"]:
         raise CLIError(resp["message"])
+
+def validate_file_path(file_path):
+    if not os.path.exists(file_path):
+        raise CLIError("File does not exist!")
+    
+def validate_file_name(file_name):
+    max_file_name_length = 110
+    if len(file_name) > max_file_name_length:
+        raise CLIError("File name should not be more than 110 characters.")
+    
+def validate_file_size(file_size):
+    max_file_size = 5242880
+    min_file_size = 1
+    if (file_size > max_file_size) or (file_size < min_file_size):
+        raise CLIError("The file must be between 1 and 5242880 bytes")
+
+def validate_file_extension(file_extension):
+    unsupported_file_extensions = [".bat", ".cmd", ".exe", ".ps1", ".js", ".vbs", ".com", ".lnk", ".reg",
+    ".bin", ".cpl", ".inf", ".ins", ".isu", ".job", ".jse", ".lnk", ".msi", ".msp", ".paf", ".pif", ".rgs", 
+    ".scr", ".sct", ".svg", ".vbe", ".vb", ".ws", ".wsf", ".wsh", ".htm", ".html"]
+    if file_extension in unsupported_file_extensions:
+        raise CLIError("The file extension is not supported.")
```

Comment:
Output the file extension being used, and suggest the file extension that should be used

Reply:
This is the same error message as Portal, so I will not be changing due to consistency. Will not fix